### ISSUE TITLE
UI: Armor Runics

### ISSUE THREAD

**Issue by jdonland** [HomebrewHomunculus/BrogueLite#31]
Current: armor of unidentified runic called "<armor type> of".
Correct: should be called "runic <armor type>" or similar

**Comment by jdonland** [comment]
Or just auto-identify?

**Comment by HomebrewHomunculus** [comment]
Yes, it sounds like a bug. IIRC, runics should be auto-identified, just like everything else. But it's been a while, I can't quite remember what the original design intention was.

There's [a new version](https://github.com/HomebrewHomunculus/BrogueCE/releases/tag/Lite-v1.10.1-RC1) over at [the new repo](https://github.com/HomebrewHomunculus/BrogueCE). I'll have to test if I can reproduce it on there.

---

This repo is now deprecated. Please see the new Brogue Lite repository at:
* https://github.com/HomebrewHomunculus
